tests/jerry/array-prototype-tostring.js

```diff
@@ -61,3 +61,10 @@ try {
   assert (e.message === "foo");
   assert (e instanceof ReferenceError);
 }
+
+// Checking behavior when undefined is passed to the function
```

Comment:
At this point not an `undefined` passed to the function. The function's `this_argument` is `undefined`.